mfs/system.go

```diff
@@ -1,12 +1,6 @@
 // package mfs implements an in memory model of a mutable ipfs filesystem.
 //
-// It consists of four main structs:
-// 1) The Filesystem
```

Comment:
This has just the short description what root is missing, but it's on the type itself. Thought that'll be one less chance of comments and code going out of sync.

Still want this back?